src/App.jsx

```diff
@@ -1,83 +1,112 @@
+import { useState } from 'react';
+import cn from 'classnames';
 import 'bulma/css/bulma.css';
+
 import './App.scss';
 
 export const goods = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
+  {
+    id: 1,
+    title: 'Dumplings',
+  },
+  {
+    id: 2,
+    title: 'Carrot',
+  },
+  {
+    id: 3,
+    title: 'Eggs',
+  },
+  {
+    id: 4,
+    title: 'Ice cream',
+  },
+  {
+    id: 5,
+    title: 'Apple',
+  },
+  {
+    id: 6,
+    title: 'Bread',
+  },
+  {
+    id: 7,
+    title: 'Fish',
+  },
+  {
+    id: 8,
+    title: 'Honey',
+  },
+  {
+    id: 9,
+    title: 'Jam',
+  },
+  {
+    id: 10,
+    title: 'Garlic',
+  },
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+export const App = () => {
+  const [selected, setSelected] = useState('Jam');
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+  return (
+    <main className="section container">
+      {selected ? (
+        <h1 className="title is-flex is-align-items-center">
+          <span>{`${selected} is selected`}</span>
+          <button
+            onClick={() => setSelected('')}
```

Comment:
```suggestion
            onClick={reset}
```
Consider moving this function from jsx